src/DataDepsGenerators.jl

```diff
@@ -110,7 +154,7 @@ function format_authors(authors::Vector)
         authors[1] * " et al."
     else
         warn("Not able to retrieve any authors")
-        "Unknown Author"
+        missing
```

Comment:
I think you were right be for,
since this is being use to typeset authors,
if no authors are given to it the correct thing is not to typeset nothing,
but to typeset `"Unknown Authors"`, or `"Authors not specified"`

I think also needed is 
`format_authors(authors::Missing) = "Authors not specified"`
So maybe for D.R.Y purposes,
specifying this as:
```
else
    warn ...
    format_authors(missing)
```